apps/prairielearn/src/pages/administratorCourseRequests/administratorCourseRequests.js

```diff
@@ -18,8 +25,35 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     res.locals.coursesRoot = config.coursesRoot;
-    const result = await sqldb.queryOneRowAsync(sql.get_requests, []);
-    _.assign(res.locals, result.rows[0]);
+    const result = await sqldb.queryRow(
+      sql.get_requests,
+      [],
+      z.object({
+        institutions: z.array(InstitutionSchema),
+        course_requests: z.array(
+          CourseRequestSchema.extend({
+            user_name: z.string(),
+            created_at: DateFromISOString.optional(),
+            approved_by: z.string().optional(),
+            approved_status: z
+              .enum(['pending', 'approved', 'denied', 'creating', 'failed'])
+              .optional(),
+            user_id: z.string().optional(),
+            jobs: z
+              .object({
+                start_date: DateFromISOString.optional(),
+                finish_date: DateFromISOString.optional(),
+                authn_user_id: IdSchema.optional(),
+                status: z.string().optional(),
+                id: IdSchema.optional(),
+                number: z.number().optional(),
+              })
+              .optional(),
+          }),
+        ),
+      }),
+    );
+    _.assign(res.locals, result);
```

Comment:
As long as we're changing this: let's do this explicitly like this instead:

```suggestion
    res.locals.institutions = result.institutions;
    res.locals.course_requests = result.course_requests;
```

You could also destructure the desired values from the query result with `const { institutions, course_requests } = await sqldb.queryRow(...)`.

Reply:
Removed the usage of lodash and destructured the the values from the query in commit [9dd4a64](https://github.com/PrairieLearn/PrairieLearn/pull/9238/commits/9dd4a647f1e05a13c5ced774f7b979c7d749fbf4).